modules/z_t_p_profiles/languages/en.json

```diff
@@ -0,0 +1,41 @@
+{
+    "Z_T_P_PROFILES": {
+        "MODULE_DETAILS": {
+            "SINGULAR_NAME": "ZTP Profile",
+            "PLURAL_NAME": "ZTP Profiles"
+        },
+        "FIELDS": {
```

Comment:
Add Japanese (ja.json), Korean (ko.json), and Simplified Chinese (zh_cn.json) JSONs for `ZTP Profiles` module